functions/Test-DbaLastBackup.ps1

```diff
@@ -259,7 +268,37 @@ Restores data and log files to alternative locations and only restores databases
 					}
 					
 					$lastbackup = Get-DbaBackupHistory -SqlServer $sourceserver -Databases $dbname -LastFull -IgnoreCopyOnly:$ignorecopyonly
-										
+
+					if($lastbackup[0].Path.StartsWith('\\') -eq $false -and $CopyDestination -eq $true) 
+					{
+						Write-Message -Level Verbose -Message "Copying backup to destination server"
+						if($destserver.BackupDirectory.StartsWith('\\') -eq $false)
+						{
+							Write-Message -Level Verbose -Message "Destination backup directory is not UNC and source does not match destination."
+							$fileexists = "Skipped"
+							$restoreresult = "Destination backup not on shared location."
+							$dbccresult = "Skipped"
+						}
+						elseif ((Test-SqlPath -SqlServer $destserver -path $destserver.BackupDirectory) -eq $true) 
+						{
+							if((Test-SqlPath -SqlServer $destserver -path ("{0}\{1}" -f $destserver.BackupDirectory,$Prefix)) -eq $false)
+							{
+								New-Item -Path ("{0}\{1}" -f $destserver.BackupDirectory,$Prefix) -ItemType Directory | Out-Null
+							}
+							Copy-Item -Path ("\\{0}\{1}" -f $lastbackup.Computername, $lastbackup.Path.replace(':','$')) -Destination ("{0}\{1}\{2}" -f $destserver.BackupDirectory,$Prefix,$lastbackup.path.split('\')[-1])
```

Comment:
I believe that $lastbackup.Path can return multiple paths. Do you have a test environment? If so, create a backup with 3 .bak files and see. I believe a foreach will be needed here.